handlers/_swagger.js

```diff
@@ -419,40 +423,52 @@ function validateRequestParameters(req, data, logger, callback) {
 
         } else if (parm.in === 'body') {
             var result = swaggerUtil.validateJSONType(parm.schema, req.body);
-
-            if (!result.valid) {
+            var polymorphicValidationErrors = [];
```

Comment:
if the result is in error, why bother with polymorphic validation?
(there could be a reason, it just needs to be explained)